go.mod

```diff
@@ -105,13 +104,12 @@ require (
 	gopkg.in/freddierice/go-losetup.v1 v1.0.0-20170407175016-fc9adea44124
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	inet.af/netaddr v0.0.0-20210704172029-d1acf45c3514
-	k8s.io/api v0.21.2
-	k8s.io/apimachinery v0.21.2
-	k8s.io/apiserver v0.21.2 // indirect; use the same version as other k8s deps
-	k8s.io/client-go v0.21.2
-	k8s.io/cri-api v0.21.2
-	k8s.io/kubectl v0.21.2
-	k8s.io/kubelet v0.21.2
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
+	k8s.io/apiserver v0.21.3 // indirect; use the same version as other k8s deps
+	k8s.io/client-go v0.21.3
+	k8s.io/cri-api v0.21.3
+	k8s.io/kubectl v0.21.3
+	k8s.io/kubelet v0.21.3
```

Comment:
It will be updated to 1.22-beta.2 in https://github.com/talos-systems/talos/pull/3935. Please wait a bit

Reply:
Also, it was already backported into 0.11.2: https://github.com/talos-systems/talos/commit/965558787734c9001e89334788c0588a580339d6

Reply:
Ok, let's merge that one, and I will rebase (and fix) #3935 after that

Reply:
yeah... I guess I get it now. The plan was to go straight to 1.22 in master.

Reply:
Yeah, it was, but 1.22 doesn't work yet :)